scripts/validate-requirements.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# bash scripts/validate-requirements.sh "END_USER_EMAIL" "ORGANIZATION_ID" "BILLING_ACCOUNT_ID"
+
+# -------------------------- Variables --------------------------
+# Expected versions of the installers
```

Comment:
Are we requiring these as minversions?

Reply:
We are looking at the version just like we have described them on the Readme. Then the compare _version function will consider every  "." separator. So, yes.

Reply:
Hmm I tried out on my local machine and it failed with TF v 1.2.0

```
Validating Terraform installation...
An incompatible Terraform version was found.
Version required is exactly 0.13.7

```

```
{
  "terraform_version": "1.2.0",
  "platform": "darwin_amd64",
  "provider_selections": {},
  "terraform_outdated": true
}
```

Reply:
Yes, and it should. Currently the expected behavior of the validation is:
- Terraform must be exact version.
- Gcloud and Git must be at least a specific version. 

So if you don't have Terraform 0.13.7 (currently configured), it will fail anyway. 
Should we change this?

Reply:
Let's pin to 1.0 now since https://github.com/terraform-google-modules/terraform-example-foundation/blob/015fe3d5716e9210cc45af9be73ae673e58df5fa/0-bootstrap/cb.tf#L20. It would be good if we can use that local as the source of truth.